src/lib/shared_utils/src/canister_specific/individual_user_template/types/hot_or_not/mod.rs

```diff
@@ -684,12 +688,12 @@ impl Post {
 
                 room_details_map.insert(global_room_id, room_detail);
                 if global_room_id.2 != ongoing_room {
-                    slot_details_map.insert(
-                        (self.id, ongoing_slot),
-                        SlotDetailsV1 {
-                            active_room_id: global_room_id.2,
-                        },
-                    );
+                slot_details_map.insert(
+                    (self.id, ongoing_slot),
+                    SlotDetailsV1 {
+                        active_room_id: global_room_id.2,
+                    },
```

Comment:
`slot_details_map` is not being updated, but is that preventing us from calculating bets? 

doesn't seem like it.

Reply:
Looked into it, slot_details_map stores active_room_id. and new rooms are created only if there are more than 100 bets and there is code in place for updating once this happens. so not an issue

Reply:
Max bets I saw were 50ish. so, that is in agreement with what I have observed.